packages/abstract-core/src/objects/module.rs

```diff
@@ -301,6 +306,7 @@ impl TryFrom<ContractVersion> for ModuleInfo {
 pub struct Module {
     pub info: ModuleInfo,
     pub reference: ModuleReference,
+    pub monetization: Monetization,
```

Comment:
I don't think monetization should be part of this struct, it should be its own thing because this way it's associated with the specific module version.

Reply:
Don't you think here is something wrong with having a configuration like this monetization object out of the general configuration ? I don't like having it as a completely separate structure. I still think we should include along with the version

Reply:
Ok, removed, we need a smart query for the monetization

Reply:
Yes it's never stored. But you were also right, it was adding useless state left and right, qi removed